File: C/2949.c
#include <stdio.h>

int main(){

    int n, a = 0, e = 0, h = 0, m = 0, x = 0;
    char nome[30], race;

    scanf("%i", &n);

    for(int i = 0; i < n; i++){
        scanf("%s %c", nome, &race);

        if(race == 'A'){
            a++;
        }else if(race == 'E'){
            e++;
        }else if(race == 'H'){
            h++;
        }else if(race == 'M'){
            m++;
        }else{
            x++;
        }
    }

    printf("%i Hobbit(s)\n", x);
    printf("%i Humano(s)\n", h);
    printf("%i Elfo(s)\n", e);
    printf("%i Anao(s)\n", a);
    printf("%i Mago(s)\n", m);

    return 0;
}

File: Python/2949.py
n = int(input())

a,e,h,m,x = 0, 0, 0, 0, 0

for i in range(n):
    name, race = input().split()
    
    if race == "A":
        a += 1
    elif race == "E":
        e += 1
    elif race == "H":
        h += 1
    elif race == "M":
        m += 1
    else:
        x += 1

print(f"{x} Hobbit(s)")
print(f"{h} Humano(s)")
print(f"{e} Elfo(s)")
print(f"{a} Anao(s)")
print(f"{m} Mago(s)")